src/core/prerender/prerender.ts

```diff
@@ -120,6 +128,11 @@ export async function prerender(nitro: Nitro) {
       }
     }
 
+    // do not prerender any files in the public assets
+    if (publicAssets.has(route)) {
```

Comment:
One thing we can optimize (for almost free -- no cache && no glob), are `nitro.options.publicAssets` with dir (no fallthrugh) such as client assets and build manifest dir.

We can also do it by default since this dirs (prefixes) were already explicitly configured as static (not mixed prefixes)

Another benefit is that regardless of prerender/copy order it works same.